specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -6166,6 +6340,122 @@
         "code",
         "message"
       ]
+    },
+    "MaintenanceWindows": {
```

Comment:
use the singular form since this is the schema for a single maintenance window - and for properties below